Client/src/main/java/org/projectcardboard/client/controller/GameController.java

```diff
@@ -206,7 +211,8 @@ private AvailableActions constructAvailableActions() {
     try {
       availableActions = new AvailableActions();
     } catch (NoSuchMethodException e) {
-      e.printStackTrace();
+      logger.warn("NoSuchMethodException was caught when 'constructAvailableActions'");
```

Comment:
Guess it'll be the same comment for a lot of these `warn` statements next to a `debug`, not going to comment on all of them individually.